src/CodeGen_D3D12Compute_Dev.cpp

```diff
@@ -870,22 +873,31 @@ void CodeGen_D3D12Compute_Dev::CodeGen_D3D12Compute_C::add_kernel(Stmt s,
             ss >> elements;
             size_t bytesize = elements * sizeof(uint32_t);
             // SM 5.1: 32KB limit for shared memory...
-            size_t packing_factor = 1;
-            while (bytesize > 32 * 1024) {
-                // must pack/unpack elements to/from shared memory...
-                elements /= 2;
-                bytesize /= 2;
-                packing_factor *= 2;
+            if (bytesize > 32 * 1024) {
+#if 0
```

Comment:
We don't want to check in `#if 0` code. If this needs to be checked-in-but-disabled, it needs some explanation about why we are keeping it around (e.g. for debugging? as an example of what not to do? etc)